### ISSUE TITLE
make test error

### ISSUE THREAD

**Issue by GrapeBaBa** [datafuselabs/datafuse#1271]
**Summary**
Env: macOS

Error:
```
test executor::action_handler_test::test_action_handler_add_database has been running for over 60 seconds
test executor::action_handler_test::test_action_handler_create_table has been running for over 60 seconds
test executor::action_handler_test::test_action_handler_do_pull_file has been running for over 60 seconds
test executor::action_handler_test::test_action_handler_drop_database has been running for over 60 seconds
test executor::action_handler_test::test_action_handler_drop_table has been running for over 60 seconds
thread panicked while panicking. aborting.
error: test failed, to rerun pass '-p fuse-store --lib'

Caused by:
  process didn't exit successfully: `/Users/kaichen/Documents/projects/datafuse/target/debug/deps/fuse_store-7b59c0e2afa8339d` (signal: 4, SIGILL: illegal instruction)
make: *** [unit-test] Error 101
```

**Comment by ariesdevil** [comment]
Known issue, for now you can try a few more times and it will works:)

**Comment by wubx** [comment]
Duplicate of #1261

**Comment by drmingdrmer** [comment]
Fixed in #1326
